ospf6d/ospf6_route.c

```diff
@@ -628,9 +630,10 @@ struct ospf6_route *ospf6_route_add(struct ospf6_route *route,
 	if (prev || next) {
 		if (IS_OSPF6_DEBUG_ROUTE(MEMORY))
 			zlog_debug(
-				"%s %p: route add %p: another path: prev %p, next %p",
+				"%s %p: route add %p: another path: prev %p, next %p node lock %u",
```

Comment:
can we call this "refcount" in debug messages instead of "lock"?

(yes it's inconsistent, i'd just like to start somewhere with moving from "lock" to "refcount" in terminology)